index.css

```diff
@@ -0,0 +1,73 @@
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+}
+
+.contact-form {
+  max-width: 800px;
+}
+
+.contact-form .cf-item {
```

Comment:
I'd recommend always using full names (so `contact-form-item`). It makes things easier to scan and understand at a glance.